control-plane-operator/hostedclusterconfigoperator/controllers/resources/konnectivity/reconcile.go

```diff
@@ -142,9 +150,23 @@ func buildKonnectivityVolumeWorkerAgentCerts(v *corev1.Volume) {
 	}
 }
 
+func buildKonnectivityVolumeCACert(v *corev1.Volume) {
+	v.ConfigMap = &corev1.ConfigMapVolumeSource{
+		DefaultMode: pointer.Int32Ptr(0640),
+	}
+	v.ConfigMap.Name = manifests.KonnectivityCAConfigMap("").Name
+}
+
 func ReconcileKonnectivityAgentSecret(secret, source *corev1.Secret) {
 	secret.Data = map[string][]byte{}
 	for k, v := range source.Data {
 		secret.Data[k] = v
 	}
 }
+
+func CopyConfigMap(cm, source *corev1.ConfigMap) {
```

Comment:
nit, this belongs in https://github.com/openshift/hypershift/tree/main/support/util